boxoffice/forms/discount.py

```diff
@@ -53,10 +68,19 @@ class CouponBasedDiscountPolicyForm(DiscountPolicyForm):
     percentage = forms.IntegerField(__("Percentage"),
         validators=[forms.validators.DataRequired(__("Please specify a discount percentage")),
             forms.validators.NumberRange(min=1, max=100, message=__("Percentage should be between >= 1 and <= 100"))])
-    discount_code_base = forms.StringField(__("Discount title"),
-        validators=[forms.validators.DataRequired(__("Please specify a discount code base")),
-        forms.validators.Length(max=20), AvailableAttr('discount_code_base', message='This discount code base is already in use. Please pick a different code base.')],
-        filters=[forms.filters.strip(), forms.filters.none_if_empty()])
+    discount_code_base = forms.StringField(
+        __("Discount title"),
+        validators=[
+            forms.validators.DataRequired(__("Please specify a discount code base")),
+            forms.validators.Length(max=20),
+            AvailableAttr(
+                'discount_code_base',
+                message='This discount code base is already in use. Please pick a different code base.'
+            ),
+            validate_unique_discount_code_base
```

Comment:
You don't need this filter if `AvailableAttr` is present. Why is it not working?